astropy/io/fits/diff.py

```diff
@@ -29,7 +29,7 @@
 from .card import Card, BLANK_CARD
 from .header import Header
 # HDUList is used in one of the doctests
-from .hdu.hdulist import fitsopen, HDUList  # pylint: disable=W0611
```

Comment:
Ok I see - indeed I think it's better the way it is now. It's a bit weird to import it in the module to use it in the docstring (and I'm surprised that the doctests are happy with that).